### ISSUE TITLE
Numerical enum fails

### ISSUE THREAD

**Issue by betim** [go-jet/jet#28]
Hi,

I just noticed that when an enum has a numerical numerator, the thing crashes quite ambiguously:
```
Generating enum sql builder files...
14:18: expected expression (and 2 more errors)
```

Also a feature request: can you add a flag say `-overwrite` which will only overwrite files and keep other things intact, for now I've commented:
```
	// fmt.Println("Cleaning up destination directory...")
	// err := utils.CleanUpGeneratedFiles(destDir)
	// utils.PanicOnError(err)
```

Cheers

**Comment by go-jet** [comment]
Hi @betim thanks for reporting the issue with generating enum files.

Regarding feature request, could you describe you use case for overwriting files. Why do you have to keep other code in generator folder?

**Comment by betim** [comment]
At first I wanted to have a controller directory inside `db-name` directory with all HTTP controllers. But now I see that it's not a good idea.

On the other hand, it would be great if there is a mode where you generate a directory for a table, say `dbn.tbl` and a folder structure would be:
```
dbn/
  tbl/
    enum.go
    model.go
    type.go
```
in this case it would be great to have the `-overwrite` flag so, you could put a `controller.go` there where you implement some business logic or the HTTP handlers.

I'm not sure if there are patterns like this, but I think this kind of separation would be quite clean.

**Comment by go-jet** [comment]
It is usually a bad practice to mix generated and custom files in the same directory. In the future there might be a feature for a user to specify folder structure, but it is not a priority now. Even then, unless there are valid use cases, generator will clean up directory before generating the files, to avoid some dangerous edge cases. For instance if the user deletes table from the database, and generator works in overwrite mode, sql builder and model files for that table will remain.

**Comment by go-jet** [comment]
Crash on generating enum SQL Builder files when database enum contains numeric values is fixed with release the [v2.2.0](https://github.com/go-jet/jet/releases/tag/v2.2.0).
Generator will construct go field name by prepending enum name to enum value, if enum value does not start with a literal.
